packages/react/src/contexts/ClerkProvider.tsx

```diff
@@ -15,24 +14,15 @@ export interface ClerkProviderProps extends IsomorphicClerkOptions {
 }
 
 function ClerkProviderBase(props: ClerkProviderProps): JSX.Element {
-  const [clerkLoaded, setClerkLoaded] = useState(false);
-
-  const clerk = useMemo(() => {
-    const { frontendApi = '', Clerk: ClerkConstructor, initialState, ...options } = props;
-    return IsomorphicClerk.getOrCreateInstance({ frontendApi, options, Clerk: ClerkConstructor, initialState });
-  }, []);
-
-  useEffect(() => {
-    void clerk.loadClerkJS().then(() => setClerkLoaded(true));
-  }, []);
+  const { initialState, children, Clerk, frontendApi, ...options } = props;
 
   return (
     <StructureContext.Provider value={StructureContextStates.noGuarantees}>
       <ClerkContextProvider
-        isomorphicClerk={clerk}
-        clerkLoaded={clerkLoaded}
+        initialState={initialState}
```

Comment:
Thats the fix 🎸

Reply:
Technically https://github.com/clerkinc/javascript/pull/225/commits/8f9481cf088c63b3cd3192cb1396596a98b11980 is needed as well!